md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/CodeUploaderAPI.java

```diff
@@ -286,4 +297,18 @@ RestWrapper checkFileExists(@PathVariable("parentProcessId") Integer parentProce
     public Object execute(String[] params) {
         return null;
     }
+
+
+    public static boolean isNumeric(String str)
+    {
+        try
+        {
+            double d = Double.parseDouble(str);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "d" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)